### PR TITLE
[ADD] Initial version of CFSSL container

### PR DESCRIPTION
**Pull request by lasley**
This brings over the CFFSL exec from clouder-community/clouder#180 & adds tests.

cc @tedsalmon @YannickB

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	&& go build -o /usr/bin/cfssl ./cmd/cfssl \
+	&& go build -o /usr/bin/cfssljson ./cmd/cfssljson \
+	&& go build -o /usr/bin/mkbundle ./cmd/mkbundle \
+	&& go build -o /usr/bin/multirootca ./cmd/multirootca \
+	&& apk del $buildDeps \
+	&& rm -rf $GOPATH/src \
+	&& echo "Build complete."
+
+# Create and Change to PKI Dir
+RUN mkdir -p /var/pki
```
Comment (tedsalmon) [inline Dockerfile:33]: `-p` is not necessarily required because `/var` will always exist but it also isn't hurting anything :)

**Review by tedsalmon — APPROVED** [review]
LGTM 👍

File: Dockerfile (hunk 1 above)
Comment (lasley) [reply Dockerfile:33]: This is also to guard from it already existing. It's good practice in the automation to use `-p`

Comment (YannickB) [reply Dockerfile:33]: Agree, saved me many times in the past during Clouder development

**Pull request merged**